inspectors/models/abstractinspectormodel.cpp

```diff
@@ -83,6 +84,9 @@ void AbstractInspectorModel::onPropertyValueChanged(const Ms::Pid pid, const QVa
         return;
 
     Ms::Score* score  = parentScore();
+    if (!score)
+        return;
```

Comment:
Is this a typical or exceptional situation? If exceptional, it is best to use
```
IF_ASSERT_FAILED(score) {
    return;   
}
```

If typical, it is better to write a comment about why there may not be score